### ISSUE TITLE
Another UnicodeDecode Error

### ISSUE THREAD

**Issue by xloem** [basak/glacier-cli#76]
```
  File "/usr/local/bin/glacier", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.6/dist-packages/glacier.py", line 729, in main
    App().main()
  File "/usr/local/lib/python3.6/dist-packages/glacier.py", line 715, in main
    self.args.func()
  File "/usr/local/lib/python3.6/dist-packages/glacier.py", line 497, in archive_upload
    file_obj=self.args.file, description=name)
  File "/usr/local/lib/python3.6/dist-packages/boto/glacier/vault.py", line 174, in create_archive_from_file
    data = file_obj.read(part_size)
  File "/usr/lib/python3.6/codecs.py", line 321, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xb7 in position 14: invalid start byte
failed                         
```

**Comment by xloem** [comment]
Reviewing my own bug report, I saw I was running python 3.  This was fixed by switching to python 2.

**Comment by soxofaan** [comment]
I think this is a duplicate of #74

**Comment by soxofaan** [comment]
@basak you can close this one too (solved by #77)

**Comment by basak** [comment]
Thanks!
